public/app/features/alerting/AlertTabCtrl.ts

```diff
@@ -140,18 +140,21 @@ export class AlertTabCtrl {
       name: model.name,
       iconClass: this.getNotificationIcon(model.type),
       isDefault: false,
+      uid: model.uid
     });
-    this.alert.notifications.push({ id: model.id });
+    if (!_.find(this.alert.notifications, { id: model.id})) {
+      this.alert.notifications.push({ uid: model.uid });
```

Comment:
@bergquist 
I left old id in settings. It's possible to overwrite them to uid, if you want.